### PR TITLE
Switch from provision to setup micromamba

### PR DESCRIPTION
**Pull request by IAlibay**
Thanks @jandom for pointing out the deprecation warning.

PR Checklist
------------
 - [x] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

----
:books: Documentation preview :books:: https://mdanalysis--4197.org.readthedocs.build/en/4197/

### CODE REVIEW

**Comment by IAlibay** [comment]
We get random non completing jobs every so often. It's unclear why but it doesn't seem related to the changes introduced here (we saw it elsewhere earlier this week).

I've reran CI a few times, it happened ~ 2/5 of times?

Seems like it always affects py3.10, maybe should look at the test it stalls on here.

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -220,14 +220,17 @@ jobs:
...
           environment-name: mda
-          extra-specs: |
-            python==${{ matrix.python-version }}
+          create-args: >-
+            python=${{ matrix.python-version }}
             pip
-          channels: jaimergp/label/unsupported-cudatoolkit-shim, conda-forge, bioconda
+          condarc: |
+            channels:
+              - jaimergp/label/unsupported-cudatoolkit-shim
```
Comment (RMeli) [inline .github/workflows/deploy.yaml:233]: I see it was already there, but worth documenting this? I had to go and search the following: https://github.com/openmm/openmm/issues/3059#issuecomment-891653746

**Review by RMeli — APPROVED** [review]

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (IAlibay) [reply .github/workflows/deploy.yaml:233]: Sure. I guess I just run in circles where everyone knows what this shim is.

File: .github/actions/setup-deps/action.yaml [hunk 2]
```diff
@@ -85,6 +85,8 @@ inputs:
     default: 'pytest-cov'
   pytest-xdist:
     default: 'pytest-xdist'
+  pytest-timeout:
```
Comment (IAlibay) [inline .github/actions/setup-deps/action.yaml]: TODO: remove once we've worked out what's slowing down CI so much

**Comment by IAlibay** [comment]
Keeping records of highest cost tests whilst I slowly increase the timeout value:

```
2023-07-26T13:57:34.4609101Z 100.00s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_creating_multiple_universe_without_offset
2023-07-26T13:57:34.4609701Z 22.76s call     MDAnalysisTests/visualization/test_streamlines.py::test_streamplot_2D
2023-07-26T13:57:34.4610190Z 14.43s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_context_manager
2023-07-26T13:57:34.4610663Z 8.83s call     MDAnalysisTests/coordinates/test_pdb.py::test_conect_bonds_all
2023-07-26T13:57:34.4611149Z 8.18s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_writer_large
2023-07-26T13:57:34.4611735Z 8.15s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u11]
2023-07-26T13:57:34.4612211Z 7.98s call     MDAnalysisTests/utils/test_pickleio.py::test_fileio_pickle
2023-07-26T13:57:34.4612681Z 6.31s call     MDAnalysisTests/coordinates/test_chemfiles.py::TestChemfiles::test_write_topology
2023-07-26T13:57:34.4613203Z 6.19s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis_weights_None
2023-07-26T13:57:34.4613712Z 6.06s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis
2023-07-26T13:57:34.4614232Z 5.88s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-26T13:57:34.4614789Z 5.81s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-26T13:57:34.4615327Z 5.64s call     MDAnalysisTests/analysis/test_encore.py::TestEncore::test_ces_error_estimation_ensemble_bootstrap
2023-07-26T13:57:34.4616078Z 5.56s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[5--2-None]
2023-07-26T13:57:34.4616619Z 5.01s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[residues]
2023-07-26T13:57:34.4617273Z 4.97s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[1-9-2]
2023-07-26T13:57:34.4617805Z 4.93s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_readers_pickle[ref_reader27]
2023-07-26T13:57:34.4618445Z 4.68s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag4]
2023-07-26T13:57:34.4619052Z 4.66s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u11]
2023-07-26T13:57:34.4619577Z 4.64s call     MDAnalysisTests/analysis/test_density.py::TestDensityAnalysis::test_run[dynamic]
2023-07-26T13:57:34.4620290Z 4.64s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[1-None-3]
2023-07-26T13:57:34.4621040Z 4.58s setup    MDAnalysisTests/coordinates/test_copying.py::test_positions_share_memory[ref_reader22]
2023-07-26T13:57:34.4621610Z 4.56s call     MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-26T13:57:34.4622171Z 4.48s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_filename[ref_reader22]
2023-07-26T13:57:34.4622753Z 4.48s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-26T13:57:34.4623357Z 4.47s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_copied_extra_attributes[ref_reader_extra_args22]
2023-07-26T13:57:34.4623948Z 4.46s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_selection
2023-07-26T13:57:34.4624619Z 4.44s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-26T13:57:34.4625190Z 4.42s call     MDAnalysisTests/converters/test_parmed.py::TestParmEdConverterPRM::test_equivalent_dihedrals
2023-07-26T13:57:34.4626754Z 4.40s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_independent_iteration[ref_reader22]
2023-07-26T13:57:34.4627342Z 4.39s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_writer_large_residue_count
2023-07-26T13:57:34.4627909Z 4.38s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_n_atoms[ref_reader22]
2023-07-26T13:57:34.4628544Z 4.36s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_next[ref_reader22]
2023-07-26T13:57:34.4629124Z 4.34s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_frame_maintained[ref_reader22]
2023-07-26T13:57:34.4629750Z 4.34s call     MDAnalysisTests/coordinates/test_chainreader.py::TestChainReaderFormats::test_set_format_tuples_and_format
2023-07-26T13:57:34.4630344Z 4.30s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[fragments]
2023-07-26T13:57:34.4630962Z 4.26s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[group]
2023-07-26T13:57:34.4631513Z 4.25s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[segments]
2023-07-26T13:57:34.4632076Z 4.24s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-26T13:57:34.4693674Z 4.21s setup    MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-26T13:57:34.4694211Z 4.14s setup    MDAnalysisTests/topology/test_altloc.py::test_write_read
2023-07-26T13:57:34.4694645Z 4.11s setup    MDAnalysisTests/topology/test_altloc.py::test_bonds
2023-07-26T13:57:34.4695075Z 4.09s setup    MDAnalysisTests/topology/test_altloc.py::test_atomgroups
2023-07-26T13:57:34.4695654Z 4.03s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag0]
2023-07-26T13:57:34.4696660Z 3.94s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter::test_Writer_dimensions[/home/runner/work/mdanalysis/mdanalysis/testsuite/MDAnalysisTests/data/lammps/datatest.data]
2023-07-26T13:57:34.4697370Z 3.90s call     MDAnalysisTests/analysis/test_align.py::TestAverageStructure::test_average_structure_ref_frame
2023-07-26T13:57:34.4697920Z 3.75s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-26T13:57:34.4698481Z 3.71s call     MDAnalysisTests/coordinates/test_chainreader.py::TestChainReaderFormats::test_set_all_formats
2023-07-26T13:57:34.4699036Z 3.61s setup    MDAnalysisTests/analysis/test_psa.py::TestPSAnalysis::test_plot_nearest_neighbors
2023-07-26T13:57:34.4699565Z 3.55s call     MDAnalysisTests/analysis/test_align.py::TestAverageStructure::test_average_structure
```

**Comment by IAlibay** [comment]
A "normal" non-failing set of test timings:

```
2023-07-28T11:37:23.1321366Z ============================= slowest 50 durations =============================
2023-07-28T11:37:23.1321909Z 21.39s call     MDAnalysisTests/visualization/test_streamlines.py::test_streamplot_2D
2023-07-28T11:37:23.1322509Z 13.25s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_context_manager
2023-07-28T11:37:23.1323076Z 8.55s call     MDAnalysisTests/coordinates/test_pdb.py::test_conect_bonds_all
2023-07-28T11:37:23.1323874Z 7.80s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_writer_large
2023-07-28T11:37:23.1324509Z 7.58s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u11]
2023-07-28T11:37:23.1325091Z 7.44s call     MDAnalysisTests/utils/test_pickleio.py::test_fileio_pickle
2023-07-28T11:37:23.1325678Z 5.97s call     MDAnalysisTests/coordinates/test_chemfiles.py::TestChemfiles::test_write_topology
2023-07-28T11:37:23.1326276Z 5.89s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis
2023-07-28T11:37:23.1326882Z 5.86s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis_weights_None
2023-07-28T11:37:23.1327545Z 5.61s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-28T11:37:23.1328207Z 5.54s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-28T11:37:23.1328865Z 5.49s call     MDAnalysisTests/analysis/test_encore.py::TestEncore::test_ces_error_estimation_ensemble_bootstrap
2023-07-28T11:37:23.1329665Z 5.06s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[5--2-None]
2023-07-28T11:37:23.1330303Z 4.78s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[residues]
2023-07-28T11:37:23.1331050Z 4.66s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[1-9-2]
2023-07-28T11:37:23.1331695Z 4.66s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_readers_pickle[ref_reader27]
2023-07-28T11:37:23.1332431Z 4.48s call     MDAnalysisTests/analysis/test_density.py::TestDensityAnalysis::test_run[dynamic]
2023-07-28T11:37:23.1333156Z 4.41s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag4]
2023-07-28T11:37:23.1334015Z 4.37s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_nonzero_start_surface[1-None-3]
2023-07-28T11:37:23.1334666Z 4.37s call     MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-28T11:37:23.1335427Z 4.32s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-28T11:37:23.1336065Z 4.30s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_selection
2023-07-28T11:37:23.1336709Z 4.29s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-28T11:37:23.1337353Z 4.20s setup    MDAnalysisTests/coordinates/test_copying.py::test_positions_share_memory[ref_reader22]
2023-07-28T11:37:23.1337974Z 4.20s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_writer_large_residue_count
2023-07-28T11:37:23.1338635Z 4.19s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_frame_maintained[ref_reader22]
2023-07-28T11:37:23.1339263Z 4.18s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u11]
2023-07-28T11:37:23.1339874Z 4.18s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_filename[ref_reader22]
2023-07-28T11:37:23.1340537Z 4.18s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_next[ref_reader22]
2023-07-28T11:37:23.1341152Z 4.17s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_n_atoms[ref_reader22]
2023-07-28T11:37:23.1341795Z 4.16s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_copied_extra_attributes[ref_reader_extra_args22]
2023-07-28T11:37:23.1342464Z 4.16s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_independent_iteration[ref_reader22]
2023-07-28T11:37:23.1343126Z 4.10s call     MDAnalysisTests/converters/test_parmed.py::TestParmEdConverterPRM::test_equivalent_dihedrals
2023-07-28T11:37:23.1343823Z 4.09s call     MDAnalysisTests/coordinates/test_chainreader.py::TestChainReaderFormats::test_set_format_tuples_and_format
2023-07-28T11:37:23.1344564Z 4.08s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[group]
2023-07-28T11:37:23.1345199Z 4.08s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[fragments]
2023-07-28T11:37:23.1345797Z 4.05s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[segments]
2023-07-28T11:37:23.1346421Z 4.04s setup    MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-28T11:37:23.1347155Z 4.00s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-28T11:37:23.1347730Z 3.86s setup    MDAnalysisTests/topology/test_altloc.py::test_atomgroups
2023-07-28T11:37:23.1348253Z 3.85s setup    MDAnalysisTests/topology/test_altloc.py::test_bonds
2023-07-28T11:37:23.1348773Z 3.84s setup    MDAnalysisTests/topology/test_altloc.py::test_write_read
2023-07-28T11:37:23.1349532Z 3.82s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter::test_Writer_dimensions[/home/runner/work/mdanalysis/mdanalysis/testsuite/MDAnalysisTests/data/lammps/datatest.data]
2023-07-28T11:37:23.1350422Z 3.75s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag0]
2023-07-28T11:37:23.1351211Z 3.69s call     MDAnalysisTests/analysis/test_align.py::TestAverageStructure::test_average_structure_ref_frame
2023-07-28T11:37:23.1351880Z 3.50s call     MDAnalysisTests/coordinates/test_chainreader.py::TestChainReaderFormats::test_set_all_formats
2023-07-28T11:37:23.1352530Z 3.49s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-28T11:37:23.1353244Z 3.41s call     MDAnalysisTests/analysis/test_align.py::TestAverageStructure::test_average_structure
2023-07-28T11:37:23.1354076Z 3.38s call     MDAnalysisTests/analysis/test_align.py::TestAverageStructure::test_average_structure_no_ref
2023-07-28T11:37:23.1354713Z 3.38s setup    MDAnalysisTests/analysis/test_psa.py::TestPSAnalysis::test_plot_nearest_neighbors
```

**Comment by IAlibay** [comment]
Another failure, a completely different test timeout:

```
2023-07-29T13:23:33.5653710Z ============================= slowest 50 durations =============================
2023-07-29T13:23:33.5654030Z 126.00s call     MDAnalysisTests/lib/test_nsgrid.py::test_issue_3183
2023-07-29T13:23:33.5654390Z 41.90s call     MDAnalysisTests/visualization/test_streamlines.py::test_streamplot_2D
2023-07-29T13:23:33.5655130Z 21.89s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[C[n+]1ccccc1-c1ccc(NC(=O)c2ccc(C(=O)Nc3ccc(-c4cccc[n+]4C)cc3)cc2)cc1 CHEMBL3230729]
2023-07-29T13:23:33.5656290Z 20.84s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[CC(C)NS(C)(=O)=Nc1ccc(-c2ccc(-c3nc4[nH]c(O[C@@H]5CO[C@@H]6[C@H](O)CO[C@H]56)nc4cc3Cl)cc2)cc1 CHEMBL4111464]
2023-07-29T13:23:33.5656840Z 18.37s call     MDAnalysisTests/coordinates/test_pdb.py::test_conect_bonds_all
2023-07-29T13:23:33.5657210Z 18.04s call     MDAnalysisTests/analysis/test_hole2.py::TestHoleAnalysis::test_context_manager
2023-07-29T13:23:33.5657610Z 15.19s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_writer_large
2023-07-29T13:23:33.5658290Z 14.74s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[c1ccc(cc1)-c1ccc(-c2ccccc2)c(-c2ccccc2)c1-c1ccccc1]
2023-07-29T13:23:33.5658820Z 14.20s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[c1ccc2c(c1)c1ccccc1c1ccccc1c1ccccc1c1ccccc21]
2023-07-29T13:23:33.5659360Z 14.16s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[C[C@@H](Oc1cc(F)ccc1Nc1ncnc2cc(N=S3(=O)CCC3)cc(F)c12)C(=O)NCC#N]
2023-07-29T13:23:33.5660100Z 13.71s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[Cc1ccc(/C(O)=C(/C([S-])=NCc2ccccc2)[n+]2cccc(CO)c2)cc1[N+](=O)[O-] CHEMBL1596426]
2023-07-29T13:23:33.5660580Z 12.18s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u11]
2023-07-29T13:23:33.5661210Z 12.15s call     MDAnalysisTests/utils/test_pickleio.py::test_fileio_pickle
2023-07-29T13:23:33.5661830Z 11.96s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis
2023-07-29T13:23:33.5662230Z 11.86s call     MDAnalysisTests/analysis/test_gnm.py::test_closeContactGNMAnalysis_weights_None
2023-07-29T13:23:33.5662620Z 11.18s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-29T13:23:33.5663340Z 11.11s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[N#Cc1c[nH]c(C(=O)Nc2ccc(-c3cccc[n+]3[O-])cc2C2=CCCCC2)n1 CHEMBL1172116]
2023-07-29T13:23:33.5663870Z 10.51s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[c1c2c(=O)n3cccc(C)c3nc2n(C)c(=N)c1C(=O)NCc1cnccc1]
2023-07-29T13:23:33.5664340Z 10.36s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-29T13:23:33.5664770Z 10.22s call     MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-29T13:23:33.5665430Z 10.04s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[c1c[nH]c(c1)-c1ccc(s1)-c1ccoc1-c1c[nH]cc1-c1ccccc1]
2023-07-29T13:23:33.5665900Z 9.87s call     MDAnalysisTests/analysis/test_encore.py::TestEncore::test_ces_error_estimation_ensemble_bootstrap
2023-07-29T13:23:33.5666320Z 9.20s call     MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-29T13:23:33.5666730Z 9.14s call     MDAnalysisTests/coordinates/test_chemfiles.py::TestChemfiles::test_write_topology
2023-07-29T13:23:33.5667130Z 9.02s call     MDAnalysisTests/analysis/test_encore.py::TestEncore::test_parallel_calculation
2023-07-29T13:23:33.5667480Z 8.99s setup    MDAnalysisTests/topology/test_altloc.py::test_atomgroups
2023-07-29T13:23:33.5667870Z 8.87s setup    MDAnalysisTests/coordinates/test_copying.py::test_positions_share_memory[ref_reader22]
2023-07-29T13:23:33.5668560Z 8.81s setup    MDAnalysisTests/topology/test_altloc.py::test_bonds
2023-07-29T13:23:33.5668910Z 8.77s setup    MDAnalysisTests/topology/test_altloc.py::test_write_read
2023-07-29T13:23:33.5669400Z 8.73s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter::test_Writer_dimensions[/Users/runner/work/mdanalysis/mdanalysis/testsuite/MDAnalysisTests/data/lammps/datatest.data]
2023-07-29T13:23:33.5669930Z 8.70s setup    MDAnalysisTests/coordinates/test_pdb.py::TestMultiPDBReader::test_conect_bonds_conect
2023-07-29T13:23:33.5670560Z 8.60s call     MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u0]
2023-07-29T13:23:33.5671030Z 8.54s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_filename[ref_reader22]
2023-07-29T13:23:33.5671420Z 8.48s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_next[ref_reader22]
2023-07-29T13:23:33.5671810Z 8.43s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_n_atoms[ref_reader22]
2023-07-29T13:23:33.5672230Z 8.41s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_initial_frame_maintained[ref_reader22]
2023-07-29T13:23:33.5672670Z 8.37s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_copied_extra_attributes[ref_reader_extra_args22]
2023-07-29T13:23:33.5673110Z 8.37s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_universe
2023-07-29T13:23:33.5673550Z 8.36s setup    MDAnalysisTests/converters/test_parmed_parser.py::TestParmedParserPSF::test_mandatory_attributes
2023-07-29T13:23:33.5673980Z 8.33s setup    MDAnalysisTests/coordinates/test_copying.py::test_reader_independent_iteration[ref_reader22]
2023-07-29T13:23:33.5674450Z 8.27s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[CC(C=CC1=C(C)CCCC1(C)C)=CC=CC(C)=CC=[NH+]C]
2023-07-29T13:23:33.5675190Z 8.26s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[Nc1nc2c(ncn2[C@H]2C[C@H](OP(=O)(O)O)[C@@H](COP(=O)(O)O)O2)c(=O)[nH]1]
2023-07-29T13:23:33.5675910Z 7.91s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_trajectory_atomgroup
2023-07-29T13:23:33.5676400Z 7.86s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag4]
2023-07-29T13:23:33.5676860Z 7.86s setup    MDAnalysisTests/parallelism/test_multiprocessing.py::test_multiprocess_COG[u10]
2023-07-29T13:23:33.5677330Z 7.83s setup    MDAnalysisTests/coordinates/test_lammps.py::TestLAMMPSDATAWriter_molecule_tag::test_molecule_tag[LAMMPSDATAWriter_molecule_tag0]
2023-07-29T13:23:33.5677850Z 7.77s call     MDAnalysisTests/converters/test_rdkit.py::TestRDKitFunctions::test_order_independant[Nc1ncnc2c1ncn2[C@H]1C[C@H](OP(=O)(O)O)[C@@H](COP(=O)(O)O)O1]
2023-07-29T13:23:33.5678320Z 7.74s call     MDAnalysisTests/coordinates/test_chainreader.py::TestChainReader::test_transform_iteration
2023-07-29T13:23:33.5678740Z 7.41s setup    MDAnalysisTests/coordinates/test_gro.py::TestGROLargeWriter::test_write_selection
2023-07-29T13:23:33.5679160Z 7.40s setup    MDAnalysisTests/transformations/test_wrap.py::test_wrap_with_compounds[fragments]
2023-07-29T13:23:33.5679480Z =========================== short test summary info ============================
```

**Comment by IAlibay** [comment]
I'm going to stick to the original intent of this PR (since it's already approved) and do the pytest-timeout stuff in a separate one.

**Comment by IAlibay** [comment]
🙃 CI stalled on the one azure runner. Going ahead with the merge anyways since we're not touching that part of CI infrastructure anyways.

**Pull request merged**